std_msgs/Header Header
bounding_box[] yolov8_boxes